### DISCUSSION TITLE
Am I able to use this to build my own taskbar?

### DISCUSSION THREAD

**Discussion by Joe23232** [fltk-rs/fltk-rs#606] [General]
Hi I am thinking of making a taskbar similar to [polybar](https://polybar.github.io/).

So I wanted to ask you using this extension, is this possible? Is it feasible as well?

**Comment by MoAlyousef** [comment]
Yes it’s both possible and feasible.

**Reply by Joe23232** [reply]
Would it work on the wayland server?

**Reply by MoAlyousef** [reply]
via xwayland

**Reply by Joe23232** [reply]
I want to avoid using xwayland as this has extra overhead as afaik it has to convert x11 to wayland in realtime.

Can't I do it directly to wayland?

**Reply by MoAlyousef** [reply]
No unfortunately. FLTK itself does its drawing via xlib on linux/bsd.

**Comment by Joe23232** [comment]
I see thanks mate :)
